### DISCUSSION TITLE
Is godoc worth it for code documentation?

### DISCUSSION THREAD

**Discussion by molikd** [hyperspike/hyperctl#3] [General] [answered]
https://pkg.go.dev/golang.org/x/tools/cmd/godoc 

Just saw godoc, looks like it does stuff like perl pods. Is it any good? good code documentation means faster startup times for new developers and better recall for seasoned veterans.

**Comment by dmolik** [comment, marked as answer]
Go Doc is available at https://pkg.go.dev/hyperspike.io/hyperctl however, it's still a work in progress.
